akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala

```diff
@@ -136,8 +136,8 @@ class CircuitBreakerSpec extends AkkaSpec with BeforeAndAfter {
           Thread.sleep(500.millis.dilated.toMillis)
         }
       }
-      within(300.millis) {
-        awaitCond(breaker().currentFailureCount == 1, 100.millis.dilated)
+      within(400.millis) {
```

Comment:
It's using the 

```
  def shortCallTimeoutCb()(implicit system: ActorSystem, ec: ExecutionContext): Breaker =
    new Breaker(new CircuitBreaker(system.scheduler, 1, 50.millis.dilated, 500.millis.dilated))
```

so technically should trigger quite fast... Shall I increase to 300ms?